users/tests/test_invitations.py

```diff
@@ -0,0 +1,11 @@
+import datetime
+from django.core import mail
+from django.urls import reverse
+from users.models import InvitationToken
+from users.invitations import generate_invitation_token
+
+
+def test_generate_invitation_token(user):
+    token = generate_invitation_token(user)
+    assert isinstance(token, InvitationToken)
+    assert token.user == user
```

Comment:
Perhaps tests the expiration date as well, as a way to ensure that the way the expiration date is created (via manager, via instance + save, etc) is correct?